Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1496,4 +1496,45 @@
     <!-- XBUT: Button text for the contact diary card button in the homescreen   -->
     <string name="contact_diary_homescreen_card_button_text">Tagebuch Führen</string>
 
+    <!-- XBUT: Button text for the contact diary card button in the homescreen   -->
+    <string name="contact_diary_onboarding_button_text">Tagebuch Führen</string>
```

Comment:
maybe move these to the diary strings file

Reply:
Please do, otherwise it will be a big mess

Reply:
These strings were added by me in the homescreen diary card ticket. I thought of putting them here because I thought since the card is part of the homescreen and not the diary stuff, it's probably better to have them in the main strings files. 

Anyway, I can see @axelherbstreith has moved them to the diary strings file already, maybe just a small observation: I think these strings should be completely removed from here and put in the (de) contact_diary_strings file. Sorry for giving you extra work!

Reply:
Done :)